### ISSUE TITLE
Code Coverage

### ISSUE THREAD

**Issue by hhenrichsen** [ShatteredSoftware/ShatteredCore#20]
Some good candidates for testing coverage are:

- [ ] Dispatch Commands 
- [ ] Dispatch Arguments 
- [ ] Many of the Persistence utilities (`Persistence`, `FileUtil`, `DataContainer`, etc.)
- [ ] Many of the Math utilities (`Vector2`, `Vector2Int`, `Vector3`, `MutableVector3Long`, `Path`, `Distribution`, etc.)
- [ ] The Adapters (`LocationAdapter`, `LocationCacheAdapter`, `Path2Adapter`)

If you are interested in any of these, please make another issue specific to that one and bring the coverage as high as possible on one of the files. I'm happy to answer any questions about the intended functionality.

**Comment by MayankMadan97** [comment]
Hey @hhenrichsen, Would it be alright if I assist with the testing?

**Comment by hhenrichsen** [comment]
> Hey @hhenrichsen, Would it be alright if I assist with the testing?

Yes. As I mentioned in the original issue, please start by opening an issue for the testing you intend to add, then create a pull request for that issue.
